pages/redirect/humanid.tsx

```diff
@@ -0,0 +1,16 @@
+import getConfig from "next/config"
+
+export async function getServerSideProps({ query, req }) {
+    const { REDIRECT_MAIN_URL } = getConfig().serverRuntimeConfig
+    const queryParamString = new URLSearchParams(query).toString()
+    return {
+        redirect: {
+            destination: `${REDIRECT_MAIN_URL}/verification?${queryParamString}`,
+            permanent: false,
+        },
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getServerSideProps` function is missing error handling. If there's an issue with the `getConfig().serverRuntimeConfig` or `new URLSearchParams(query).toString()`, it could cause the application to crash. Consider adding a try-catch block to handle potential errors.
```diff
+    try {
-        const { REDIRECT_MAIN_URL } = getConfig().serverRuntimeConfig
-        const queryParamString = new URLSearchParams(query).toString()
+        return {
+            redirect: {
+                destination: `${REDIRECT_MAIN_URL}/verification?${queryParamString}`,
+                permanent: false,
+            },
+        }
+    } catch (error) {
+        // Handle error appropriately, e.g., log it and return a meaningful error message
+        console.error(error);
+        return {
+            props: {},
+            error: 'An error occurred while processing the request.'
+        }
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->